src/app/shared/classes/test-flow-base-pages/waiting-room-to-car/waiting-room-to-car-base-page.ts

```diff
@@ -217,7 +220,19 @@ export abstract class WaitingRoomToCarBasePageComponent extends BasePageComponen
   }
 
   async onViewTestCentreJournal(): Promise<void> {
-    await this.router.navigate([TEST_CENTRE_JOURNAL_PAGE]);
+    if (this.isEndToEndPracticeMode) {
+      await this.presentAlert();
+    } else await this.router.navigate([TEST_CENTRE_JOURNAL_PAGE]);
+  }
```

Comment:
Pedantic i guess, but DES code often favours returning and then removing the else i.e.
```
async onViewTestCentreJournal(): Promise<void> { 
   if (this.isEndToEndPracticeMode) {
        await this.presentAlert();
        return;
   }
  await this.router.navigate([TEST_CENTRE_JOURNAL_PAGE]);
}
```

Reply:
its temporary code until a back log story is delivered but agree, will ammend to the standard